torch/onnx/symbolic_opset11.py

```diff
@@ -45,6 +45,12 @@ def _cast_if_not_none(tensor, dtype):
     return g.op("Clip", self, min, max)
 
 
+# Opset 11 gather accepts negative indices
+@parse_args('v', 'i', 'v')
+def select(g, self, dim, index):
```

Comment:
the description of the PR says the export is uupdated for index_select opset 11. Is there a similar change for index_select required or was it a typo in the PR description?